### ISSUE TITLE
JetBrains IDEA support

### ISSUE THREAD

**Issue by jDekx** [TehLeo/junion#5]
Any plans of development a plugin for JetBrains IDEA?

**Comment by TehLeo** [comment]
Yes, I will get back to plugins and integration once I release the next version. Currently gradle can be setup within IDEA to compile with JUnion, but checking if I can develop the plugin for IDEA is on the todo list.

**Comment by Scoppio** [comment]
I'll add to the request for the Idea support.
I just broke my entire project trying to compile it manually

**Comment by TehLeo** [comment]
I'm sorry to hear that. I hope you do have backups of your projects. 
Once I'll find some time to work a bit on this project, I might check if I can implement IDEA integration.
